<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LpWidget</class>
 <widget class="QWidget" name="LpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Графический метод решения задачи линейного программирования</string>
  </property>
  <widget class="QLabel" name="funcLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Целевая функция (F = ax1 + bx2):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="funcEditX1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="placeholderText">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLabel" name="c1Label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ограничение 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="funcEditX2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="placeholderText">
    <string>b</string>
   </property>
  </widget>
  <widget class="QComboBox" name="funcPurposeBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>45</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>max</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>min</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="x1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 +</string>
   </property>
  </widget>
  <widget class="QLabel" name="x2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint1EditX1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="placeholderText">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint1EditValue">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="placeholderText">
    <string>c</string>
   </property>
  </widget>
  <widget class="QLabel" name="x2_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint1EditX2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="placeholderText">
    <string>b</string>
   </property>
  </widget>
  <widget class="QComboBox" name="constraint1SignBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>115</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="x1_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 +</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint2EditX2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="placeholderText">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint2EditX1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="placeholderText">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint2EditValue">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="placeholderText">
    <string>c</string>
   </property>
  </widget>
  <widget class="QComboBox" name="constraint2SignBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>185</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="c1Label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ограничение 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="x1_3">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 +</string>
   </property>
  </widget>
  <widget class="QLabel" name="x2_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>190</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint3EditX2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="placeholderText">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint3EditX1">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="placeholderText">
    <string>a</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="constraint3EditValue">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="placeholderText">
    <string>c</string>
   </property>
  </widget>
  <widget class="QComboBox" name="constraint3SignBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>255</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>&gt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>&lt;=</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="c1Label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ограничение 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="x1_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>260</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x1 +</string>
   </property>
  </widget>
  <widget class="QLabel" name="x2_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>260</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="solveButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>271</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Решить задачу и построить график</string>
   </property>
  </widget>
  <widget class="QSlider" name="targetSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="scaleSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>488</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ansLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>здесь будет ответ</string>
   </property>
  </widget>
  <widget class="QLabel" name="funcLabel_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>380</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Значение целевой функции</string>
   </property>
  </widget>
  <widget class="QLabel" name="funcLabel_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>458</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Масштаб</string>
   </property>
  </widget>
  <widget class="QLabel" name="funcLabel_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>530</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ</string>
   </property>
  </widget>
  <widget class="QLabel" name="minTargetLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>412</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="minScaleLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxTargetLabel">
   <property name="geometry">
    <rect>
     <x>943</x>
     <y>412</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="maxScaleLabel">
   <property name="geometry">
    <rect>
     <x>943</x>
     <y>490</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="curTargetLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>435</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="curScaleLabel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>510</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>funcEditX1</tabstop>
  <tabstop>funcEditX2</tabstop>
  <tabstop>funcPurposeBox</tabstop>
  <tabstop>constraint1EditX1</tabstop>
  <tabstop>constraint1EditX2</tabstop>
  <tabstop>constraint1SignBox</tabstop>
  <tabstop>constraint1EditValue</tabstop>
  <tabstop>constraint2EditX1</tabstop>
  <tabstop>constraint2EditX2</tabstop>
  <tabstop>constraint2SignBox</tabstop>
  <tabstop>constraint2EditValue</tabstop>
  <tabstop>constraint3EditX1</tabstop>
  <tabstop>constraint3EditX2</tabstop>
  <tabstop>constraint3EditValue</tabstop>
  <tabstop>solveButton</tabstop>
  <tabstop>constraint3SignBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
